filters/f_lavfi.c

```diff
@@ -981,9 +981,16 @@ static bool is_usable(const AVFilter *filter, int media_type)
     int nb_inputs  = avfilter_pad_count(filter->inputs),
         nb_outputs = avfilter_pad_count(filter->outputs);
 #endif
-    return nb_inputs == 1 && nb_outputs == 1 &&
-           avfilter_pad_get_type(filter->inputs, 0) == media_type &&
-           avfilter_pad_get_type(filter->outputs, 0) == media_type;
+    bool ret = false;
+    if (nb_inputs == 1 && nb_outputs == 1) {
+        ret = avfilter_pad_get_type(filter->inputs, 0) == media_type &&
+              avfilter_pad_get_type(filter->outputs, 0) == media_type;
+    } else {
+        if ((nb_inputs != 1 && filter->flags & AVFILTER_FLAG_DYNAMIC_INPUTS) ||
+            (nb_outputs != 1 && filter->flags & AVFILTER_FLAG_DYNAMIC_OUTPUTS))
+            ret = true;
+    }
```

Comment:
I think this already supports only one of them being dynamic? Sorry if I'm having a brainfade here. 

I tested libplacebo and it connects, as well as works and libplacebo only has dynamic input, not output.

Reply:
> I think this already supports only one of them being dynamic?

But doesn't check the other one in this case.

Consider 
`nb_inputs == 0 and dynamic`
`nb_outputs == 2 and not dynamic`
it will allow it, while it should reject output

Reply:
ah! thanks I'll fix it